test/tests/ports/transaction/checkGasPrice.spec.ts

```diff
@@ -70,14 +93,70 @@ test('checkGasPrice component', function ({ components }) {
656d616c65000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000006636f6d6d6f6e0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002b313a773a4e65772073746172743a3a746f705f686561643a426173654d616c652c4261736546656d616c6500000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000006636f6d6d6f6e0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002b313a773a4e65772073746172323a3a746f705f686561643a426173654d616c652c4261736546656d616c6500000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000006756e697175650000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002a313a773a556e7469746c65643a3a746f705f686561643a426173654d616c652c4261736546656d616c650000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000006636f6d6d6f6e0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002a313a773a41726d61747572653a3a746f705f686561643a426173654d616c652c4261736546656d616c650000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000800000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000c00000000000000000000000000000000000000000000000000000000000000006636f6d6d6f6e0000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000002c313a773a41726d617475726532323a3a746f705f686561643a426173654d616c652c4261736546656d616c65000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000000',
```

Comment:
In these cases, as the test data isn't relevant (we can't parse the data with ease mentally), it might be better to move it to a different file so it doesn't pollute the tests, what do you think?